tests/unittest/querybindblocker_ratelimit_table_empty/main_test.go

```diff
@@ -0,0 +1,133 @@
+package main
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/paypal/hera/tests/unittest/testutil"
+	"github.com/paypal/hera/utility/logger"
+)
+
+var mx testutil.Mux
+
+func cfg() (map[string]string, map[string]string, testutil.WorkerType) {
+	fmt.Println("setup() begin")
+	appcfg := make(map[string]string)
+	// best to chose an "unique" port in case golang runs tests in paralel
+	appcfg["bind_port"] = "31002"
+	appcfg["log_level"] = "5"
+	appcfg["log_file"] = "hera.log"
+	appcfg["rac_sql_interval"] = "0"
+	appcfg["enable_query_bind_blocker"] = "true"
+
+	opscfg := make(map[string]string)
+	opscfg["opscfg.default.server.max_connections"] = "3"
+	opscfg["opscfg.default.server.log_level"] = "5"
+	if os.Getenv("WORKER") == "postgres" {
+		return appcfg, opscfg, testutil.PostgresWorker
+	}
+	return appcfg, opscfg, testutil.MySQLWorker
+}
+
+func teardown() {
+	mx.StopServer()
+}
+
+func TestMain(m *testing.M) {
+	os.Exit(testutil.UtilMain(m, cfg, nil))
+}
+
+func executeQuery(t *testing.T, db *sql.DB) {
+	ctx := context.Background()
+	conn, err := db.Conn(ctx)
+	defer conn.Close()
+	if err != nil {
+		t.Fatalf("conn %s", err.Error())
+	}
+
+	tx, _ := conn.BeginTx(ctx, nil)
+	stmt, err := tx.PrepareContext(ctx, "/*qbb_test.find*/select id, note from qbb_test where id=? for update")
+	defer stmt.Close()
+	if err != nil {
+		t.Fatalf("Error prep sel %s\n", err.Error())
+		tx.Rollback()
+	}
+	_, err = stmt.Query(11)
+	if err != nil {
+		t.Fatalf("Failed to execute query with error: %v", err)
+		tx.Rollback()
+	}
+	tx.Commit()
+	stmt.Close()
+	conn.Close()
+}
+
+func TestQueryBindBlockerTableNotExistOrEmpty(t *testing.T) {
+	testutil.RunDML("DROP TABLE IF EXISTS hera_rate_limiter")
+	testutil.RunDML("DROP TABLE IF EXISTS qbb_test")
+	testutil.RunDML("create table qbb_test (id numeric, note varchar(111))")
```

Comment:
We don't need this table (qbb_test) for this test. We can simplify this test by removing the other lines that are not required. 

1. Initially, we could validate for the table missing error.
2. Sleep 
3. Validate table read.

Reply:
Incorporated review comments